src/main/scala/s2d/core/Cursor.scala

```diff
@@ -1,11 +1,11 @@
-package S2D.core
-
-import gl.GL.*
-import gl.GLExtras.*
-import S2D.types.*
-import sdl2.SDL.*
-import sdl2.Extras.*
-import S2D.core.Window.*
+package s2d.core
```

Comment:
Notes on the directory structure:

1. All the lib code is now in a folder called `src`
2. There is a parallel structure for lib code (`main`) and testing code (`test`).
3. There are no tests currently, but I've added a `.gitkeep` to keep the folder in place.
4. The main `project.scala` is still in the root.